static/bidder-info/iionads.yaml

```diff
@@ -0,0 +1,16 @@
+endpoint: "http://{{.Host}}.iionads.com/openrtb/{{.PublisherID}}"
```

Comment:
should this be `https` ? similar thoughts for `static/bidder-info/limelightDigital.yaml` endpoint

Reply:
As I can see, it is just not recommended, but it's not prohibited. As I see, a lot of Prebid Server adapters do the same.

Reply:
Is it blocking the pull request merge?

Reply:
Although not prohibited now.  But if possible should prefer using `https` over `http`.  Let us know this change is possible. If not then we can merge PR change.

Reply:
We use `http` for server-to-server communication. Please merge the pull request as it is, if it is possible :-)

Reply:
HTTP is tolerated for server-to-server. It's understood that it's more expensive to run load balancers that process SSL.
Client-side adapters must use HTTPS.

Reply:
@bretg You are absolutely right, so we use `https` in our client side Prebid.js adaper https://github.com/prebid/Prebid.js/blob/master/modules/limelightDigitalBidAdapter.js#L119